lace/serialization/obj/__init__.py

```diff
@@ -85,10 +85,17 @@ def path_relative_to_mesh(filename):
             with s3.open(materials_filepath, 'r') as f:
                 mesh.materials_file = f.readlines()
             mesh.materials_filepath = materials_filepath
+
     if hasattr(mesh, 'materials_file'):
-        for line in mesh.materials_file:
-            if line and line.split() and line.split()[0] == 'map_Ka':
-                mesh.texture_filepath = path_relative_to_mesh(line.split()[1])
+        mesh.materials = dict([
+            (line.split()[0], ' '.join(line.split()[1:]))
+            for line in mesh.materials_file if line.split()
+        ])
```

Comment:
I initially used `line.split(None, 1)` as you suggest but when that gave me values like `['ka', '0.329412 0.223529 0.027451\r\n']`, I figured it looked nicer the other way. I also had an intermediate point where the value remained a list but that just had its own downsides.

You are right, though, about filtering for only `map_K` lines. I was trying to parse the file more than I really needed for this particular task.